main.go

```
@@ -1,17 +1,33 @@
+	if err != nil {
+		panic("failed to connect database")
+	}
+
+	db.AutoMigrate(&models.Feature{})
+
+	r := chi.NewRouter()
+
+	r.Use(middleware.Logger)
+
+	r.Get("/", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("Hello World!"))
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)